docs/source/using/interacting-with-openshift.adoc

```diff
@@ -160,6 +161,23 @@ To change the property value and allow cross-origin requests from all domains, r
 $  minishift openshift config set --patch '{"corsAllowedOrigins": [".*"]}'
 ----
 
+[NOTE]
+====
+If you get the error `The specified patch need to be a valid JSON.` when you run the above command,
```

Comment:
👍 nice. This describes the problem in a generic way without generalising it for a specific OS/shell. And it also provides two concrete examples.